Source/engine/render/blit_impl.hpp

```diff
@@ -64,6 +65,37 @@ struct BlitWithMap {
 	}
 };
 
+DVL_ALWAYS_INLINE DVL_ATTRIBUTE_HOT void BlitFillWithLightmap(uint8_t *dst, unsigned length, uint8_t color, const Lightmap *lightmap)

```

Comment:
`lightmap` cannot be null so it should be passed by reference

Reply:
`world_draw_black_tile()` passes `nullptr` for the `lightmap` argument into `RenderLeftTriangle<LightType::FullyDark, ...>()`.

Reply:
That's true. These blit functions would never see the `nullptr` value.